app/views/home/PageHome.vue

```diff
@@ -250,6 +250,15 @@ export default Vue.extend({
     BaseCloudflareVideo
   },
   data () {
+    let teacherSignupModal = true
+    let teacherModalPath = 'teacher'
+    let teacherLink = null
+    if (me.isTeacher()) {
+      teacherLink = 'https://codecombat.com/teachers/classes'
```

Comment:
I think we can leave off the domain, so that it stays on whatever domain it's currently on.
```suggestion
      teacherLink = '/teachers/classes'
```